### ISSUE TITLE
[Request] Update code to work with new API

### ISSUE THREAD

**Issue by pascalo7** [omegavesko/DiscordAnimeStatus#4]
The discord API got an update and so did Discord.net, could you update the program to work with it or is this project no longer being maintained?

**Comment by lynnntropy** [comment]
I honestly haven't touched this code in a decent amount of time, but I'm not opposed to the idea of updating (or rewriting) it to keep up with the API. I'll definitely have a look at it whenever I have the time.

**Comment by pascalo7** [comment]
If you do decide to continue developing it you should check out the rich persistence stuff that has been added in the past few months. You can do stuff like this with it and a ton more
![image](https://user-images.githubusercontent.com/2935785/31864579-84c94d44-b757-11e7-9f55-f5f1dc02bfb5.png)

**Comment by lynnntropy** [comment]
[Rich Presence](https://discordapp.com/developers/docs/topics/rich-presence) is currently in a private alpha phase, and only available to select game developers. It's not something you or I can really play with at this point, or potentially ever, depending on how strict they decide to be about letting people use this feature for non-game purposes.

**Comment by pascalo7** [comment]
I've seen a few posts on /r/discordapp/ of people setting it to show that they are watching the bee movie and just making bee movie memes with the rich persistence so it must be more public that it seems.

![image](https://user-images.githubusercontent.com/2935785/31864944-541283ee-b75e-11e7-90e6-a9654b0f55f3.png) 
![image](https://user-images.githubusercontent.com/2935785/31864945-5653d7b6-b75e-11e7-91b9-35d9271a4fc2.png)

**Comment by lynnntropy** [comment]
You're right, it does look like it's possible to do this with a selfbot if you know what the API expects to get. Guess it's worth looking into.

**Comment by lynnntropy** [comment]
Keeping in mind that I'm not sure I _want_ to release any code that uses this feature before it's considered a little more widely available. I'm not one of those people who think Discord actually bans people for using selfbots, but I wouldn't want to get anyone (or myself) in hot water for playing with something I'm not supposed to.
